app/bundles/InstallBundle/Configurator/Step/CheckStep.php

```diff
@@ -22,50 +22,35 @@ class CheckStep implements StepInterface
 {
     /**
      * Flag if the configuration file is writable.
-     *
-     * @var bool
      */
     private $configIsWritable;
 
     /**
      * Path to the kernel root.
-     *
-     * @var string
      */
     private $kernelRoot;
 
-    /**
```

Comment:
Is there a reason for these being removed?

Reply:
But this is not issue.
It's standard feature of CS fixer
It use param **no_superfluous_phpdoc_tags** what based on documentation https://cs.symfony.com/ works properly If remove this comments

@escopecz @alanhartless can you look on it, because for me it's really paint-full and time wasting

Reply:
CS fixer make it. 
Option **no_superfluous_phpdoc_tags**  make it. Based on docs https://cs.symfony.com/
> Removes @param, @return and @var tags that don’t provide any useful information.

@alanhartless @escopecz We need know what now. Because it's annoying take it back during each commit, waste a lot of time with it.

Reply:
It looks like CS Fixer is not doing this. How can it know what type is `$site_url`?

Reply:
I returned the var types back but I don't have permission to push it to this PR. @madmath03 did you disable contributions from core members for this PR?

Reply:
@escopecz It does. If  I set no_superfluous_phpdoc_tags to false, then CS fixer not remove it. But also not remove it on __construct - see this PR https://github.com/mautic/mautic/pull/8981
Maybe some bug in CS fixer, probably related to PHP 7.4

Tomas wrote in his blog (https://tomasvotruba.com/blog/2020/07/02/how-to-patch-package-in-vendor-yet-allow-its-updates/)-  But since PHP 7.4, we could drop the @var annotation
I am using PHP 7.2, I've disabled CS fixer pre commit at the moment, but it's really annoying.

Reply:
> I returned the var types back but I don't have permission to push it to this PR. @madmath03 did you disable contributions from core members for this PR?

@escopecz I did not do that, it simply is that GitHub forbids PR push from forks owned by an Organization (I made those improvements for my company).

Right now, the only solution is to use GitHub "_code suggestion_" when doing code review.